internal/app/app.go

```diff
@@ -60,14 +67,23 @@ func (a *App) Run(ctx context.Context) error {
 	// start messenger (receiver)
 	messenger, userIDUpdatedChan := a.startMessenger(gCtx, g)
 
-	// initiate tvm client
-	tvmClient, err := tvm.NewClient(a.cfg.TvmConfig) // TODO make client init conditional based on provided config
+	client, err := evm.NewClient(a.cfg.EvmConfig) // TODO make client init conditional based on provided config
```

Comment:
```suggestion
	evmClient, err := evm.NewClient(a.cfg.EvmConfig) // TODO make client init conditional based on provided config
```
so it will be clear down the line, what exactly this client is

Reply:
also, is TODO still relevant?